x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/search_index/search_index_details_page.tsx

```diff
@@ -0,0 +1,153 @@
+import { DeleteIndexModal } from '../../index_delete_modal';
+
+export const SearchIndexDetailsPage: FunctionComponent<
+  RouteComponentProps<{ indexName: string }>
+> = ({ location: { search }, history }) => {
+  const queryParams = useMemo(() => new URLSearchParams(search), [search]);
+  const indexName = queryParams.get('indexName') ?? '';
+  const {
+    indexLoadingError,
+    index,
+    isIndicesLoading,
+    isDeleteLoading,
+    fetchIndexDetails,
+    deleteIndex,
+    navigateToIndicesList,
+  } = useIndexFunctions(indexName, search, history);
+
+  const [isShowingMoreOptionsPopover, setShowMoreOptionsPopover] = useState<boolean>(false);
+  const [isShowingDeleteModal, setShowDeleteModal] = useState<boolean>(false);
+
+  useEffect(() => {
+    fetchIndexDetails();
+  }, [fetchIndexDetails]);
+
+  const pageTitle = <>{index?.name}</>;
+
+  if (!indexName) {
+    return <DetailsPageEmptyIndexNameError />;
+  }
+  if ((isIndicesLoading || isDeleteLoading) && !index) {
+    return <DetailsPageLoading />;
+  }
+  if (indexLoadingError || !index) {
+    return (
+      <DetailsPageError
+        indexName={indexName}
+        resendRequest={fetchIndexDetails}
+        navigateToIndicesList={navigateToIndicesList}
+      />
+    );
+  }
+
+  return (
+    <>
+      <EuiPageSection paddingSize="none">
+        <EuiButton
+          data-test-subj="searchIndexDetailsBackToIndicesButton"
+          color="text"
+          iconType="arrowLeft"
+          onClick={navigateToIndicesList}
+        >
+          <FormattedMessage
+            id="xpack.idxMgmt.searchIndexDetails.backToIndicesButtonLabel"
+            defaultMessage="Back to indices"
+          />
+        </EuiButton>
+      </EuiPageSection>
+      <EuiSpacer size="l" />
+      <EuiPageHeader
+        data-test-subj="searchIndexDetailsHeader"
+        pageTitle={pageTitle}
+        rightSideItems={
+          isIndicesLoading || isDeleteLoading
+            ? []
+            : [
+                <EuiFlexGroup>
+                  <EuiFlexItem>
+                    <EuiPopover
+                      isOpen={isShowingMoreOptionsPopover}
+                      button={
+                        <EuiButtonIcon
+                          data-test-subj="searchindexDetailsMoreOptionsButton"
+                          data-telemetry-id="idxMgmt-searchIndexDetails-moreOptionsButton"
+                          color="primary"
+                          size="m"
+                          iconType="boxesVertical"
+                          aria-label={i18n.translate(
+                            'xpack.idxMgmt.searchIndexDetails.moreOptionsButton.ariaLabel',
+                            {
+                              defaultMessage: 'More options',
+                            }
+                          )}
+                          onClick={() => setShowMoreOptionsPopover(!isShowingMoreOptionsPopover)}
+                        />
+                      }
+                    >
```

Comment:
q: is it me or does this MenuItem ends up having a looot of padding? 🤔 

![Screenshot 2024-08-27 at 09 00 36](https://github.com/user-attachments/assets/0dd8cf27-67ee-43b6-9358-8155516f4d9c)

Reply:
Agreed. I think we can just add `panelPaddingSize` set to 'none' ([EUI link](https://eui.elastic.co/#/layout/popover))

And I don't think the surrounding `EuiFlexGroup` is needed here.  I believe the `EuiPageTemplate.Header` just needs an array of items and it automatically drops them inline with default spacing.